### PR TITLE
Rapsearch

### PR DESCRIPTION
**Pull request by HenningTimm**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by HenningTimm** [comment]
@bioconda/core The last recipe I added was some time ago so I am not sure about the protocol. All tests check out, do I need to do anything else before I can (squash and) merge this PR? Am I free to merge a PR myself if all tests work and all other guidelines are met?

**Comment by daler** [comment]
Thanks @HenningTimm, new docs are here https://bioconda.github.io/contributing.html and in particular see the [checklist](https://bioconda.github.io/guidelines.html#bioconda-recipe-checklist). The current protocol is pretty much 1. make sure recipe satisfies the checklist (it does) and 2. make sure tests pass (they do). So you're good to merge when you're ready.

Not sure we have a policy on squashing merges or not, so whatever you prefer.

**Comment by HenningTimm** [comment]
Thanks for the quick feedback @daler.

File: recipes/rapsearch/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    - libgcc
+
+test:
+  commands:
+    - rapsearch 2>&1 | grep rapsearch
+    - prerapsearch 2>&1 | grep "Usage\:"
+
+about:
+  home: http://omics.informatics.indiana.edu/mg/RAPSearch2/
+  license: GNU General Public License
```
Comment (johanneskoester) [inline recipes/rapsearch/meta.yaml]: Can you please add the actual GPL version? 2 or 3?

Comment (johanneskoester) [inline recipes/rapsearch/meta.yaml]: In that case, I think it is fine.

**Pull request merged**